packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,361 @@
+  categoryRoute={SidebarCategoryRoutes.EXPERIENCES}
+/>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Base principles
+
+- Do not duplicate Console content in different places
+- Focus on the user journey, not just your product or feature
+- Rely on existing patterns before creating a new one
+- Use Paste with the Default theme for all new content
+- Console navigation is optimized for findability, not promoting products or features
+
+## Develop vs. Monitor
+
+In our research, we found that a majority of users expect content to be organized by
+the phases of their development lifecycle, not by product vertical.
+A user's journey in building an application on Twilio starts with **Developing**,
+then moves into **Monitoring** after they launch their application.
+
+<Box display={['block', 'flex']} columnGap="space40" alignItems="stretch">
+  <Box width={['100%', '50%']} marginBottom="space20">
+    <Card>
+      <Paragraph>
+        <strong>Develop</strong>
+      </Paragraph>
+      <Paragraph>
+        The customer is getting started, learning about how Twilio works, testing out Twilio product(s), prototyping a
+        POC, buying and configuring numbers, setting up services, building their application, and integrating it into
+        their own system.
+      </Paragraph>
+      <Paragraph>Primary Personas: Developers, Business and Product leaders</Paragraph>
+    </Card>
+  </Box>
+  <Box width={['100%', '50%']} marginBottom="space20">
+    <Card>
+      <Paragraph>
+        <strong>Monitor</strong>
+      </Paragraph>
+      <Paragraph>
+        The customer has "launched" their application into production and needs to monitor it's activity, coming back to
+        Console to troubleshoot, debug, review logs and insights.
+      </Paragraph>
+      <Paragraph>Primary Personas: Customer support, Developers, and Product Managers</Paragraph>
+    </Card>
+  </Box>
+</Box>
+<br />
+<br />
+
+When building your product, think about the journey your users will take.
+Within **Develop**, a user should be able to accomplish all tasks related to learning about the product,
+trying it out, developing and managing services, and applying product settings.
+Within **Monitor**, a user should be able to find all content related to monitoring their application,
+such as logs and insights.
+
+On a rare occasion, there are tasks that span between Develop and Monitor.
+For example, a user may learn that calls are not connecting in India through Voice Insights (in **Monitor**),
+and that they need to turn on India in their Voice Geo-permissions (in **Develop**). In cases like these,
+good interlinking between the relevant pages or contextual quick actions will provide a streamlined
+user experience. In addition, the tabs will hold the menu states so that less clicks are required to drill
+in and between menus under each tab.
+
+![](../../../assets/images/experiences/console/develop-or-monitor.png)
```

Comment:
question: all of these images are showing up blurry to the point that I can't really read the small text, do we plan to address that by wrapping them in anchors, or are the img captions enough? looks like some currently are clickable and others aren't, but the ones that are open in a new tab. might be better to open those in a new tab